python/build_ceed_cffi.py

```diff
@@ -13,6 +13,27 @@
 ceed_version_ge = re.compile(r'\s+\(!?CEED_VERSION.*')
 
 
+# Checks to see if a c line is part of the lines we have to exclude (macros)
+def is_valid_line(line):
+    if (line.startswith("#") and not line.startswith("#include")):
+        return False
+    if (line.startswith("  CEED_QFUNCTION_ATTR")):
+        return False
+    if (line.startswith("  static")):
+        return False
```

Comment:
```suggestion
    if (line.startswith("  static const char")):
        return False
```
ah, lets be more specific then